cmake/compiler_setup.cmake

```diff
@@ -116,14 +116,6 @@ if(MSVC)
   )
 endif()
 
-foreach(flag ${common_flags} ${c_flags})
```

Comment:
What's the reason to delete these lines?

Reply:
These change global state and apply our preferences for flags to all compile actions. This breaks `firebase-cpp-sdk` and its dependencies which don't hold themselves to anywhere near as high a standard.